DFC.App.FindACourse/Controllers/DetailsController.cs

```diff
@@ -31,7 +33,7 @@ public IActionResult Details(string courseId, string runId)
 
             try
             {
-               model.courseDetails = this.findACourseService.GetCourseDetails(courseId, runId).Result;
+               model.CourseDetails = this.findACourseService.GetCourseDetails(courseId, runId).Result;
             }
             catch (Exception ex)
```

Comment:
Also when you this exception its swallowing it and logging it as information.

Reply:
I've changed this to log error. Good spot.